app/src/main/res/values/strings.xml

```diff
@@ -4,10 +4,17 @@
     <string name="search">Search</string>
     <string name="favorite">Favorite</string>
     <string name="profile">Profile</string>
+    <string name="name_movie">Name movie</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.name_movie` appears to be unused
 Unused resources make applications larger and slow down builds.